lib/convert.ml

```diff
@@ -0,0 +1,58 @@
+module T = Types
```

Comment:
I think the whole `Convert` module could simply be moved to `Intermed`, having `a from_type_declaration` function in `Intermed.Type_decl`.